ansible_ai_connect/ai/api/views.py

```diff
@@ -134,6 +131,55 @@
 }
 
 
+class OurAPIView(APIView):
```

Comment:
Is it better to extend `GenericAPIView`?

`GenericAPIView` extends `APIView` and already has a `serializer_class` attribute

Reply:
Yes, maybe. I will take a look once this PR is merged.

Reply:
So, answering a bit late on this. I'm not so sure `GenericAPIView` comes with various new attributes we don't need and if we want to use the `serializer_class` attribute, we need to also overload some existing classes.
My conclusion is both work and that we're better off with just `APIView` since it's the most basic one.